conda/install.py

```diff
@@ -227,6 +226,10 @@ def warn_failed_remove(function, path, exc_info):
 
 def exp_backoff_fn(fn, *args):
     """Mostly for retrying file operations that fail on Windows due to virus scanners"""
+    if not on_win:
+        return fn(*args)
+    import random
```

Comment:
@mingwandroid Proceeding with making this windows-only right now. Already scheduled some time with you tomorrow to discuss further.